MongoDB/containers/0.1.0/mongodb-config/connect.sh

```diff
@@ -1,13 +1,16 @@
 #!/bin/bash
-DIG=/opt/rancher/bin/dig
+
+set -x
```

Comment:
typically we don't deploy with -x.. if you want it, maybe put it behind a debug env var or something. CATTLE_SCRIPT_DEBUG I think is the convention...